src/bin/uudoc.rs

```diff
@@ -196,15 +196,7 @@ impl<'a, 'b> MDWriter<'a, 'b> {
     }
 
     fn description(&mut self) -> io::Result<()> {
-        if let Some(after_help) = self.markdown_section("about") {
-            return writeln!(self.w, "\n\n{}", after_help);
-        }
-
-        if let Some(about) = self
-            .command
-            .get_long_about()
-            .or_else(|| self.command.get_about())
-        {
+        if let Some(about) = self.command.get_about() {
```

Comment:
I'd rather keep this coming from the markdown directly. That enables us to do some cool markdown things.

(Currently only backticks, which are removed before we pass them to clap, but render as inline code blocks in the online docs. Full compile-time markdown support is one of the things I'm working on in supporting for the new argument parser.)

Reply:
See this stub of a function: https://github.com/uutils/coreutils/blob/bc0273f48feac15f06238f2433745f4480bb5c0a/src/uucore_procs/src/lib.rs#L52-L54

Also I see we aren't using that function in about, so that's another thing I'll have to look into.

Reply:
I suppose we could skip it for now if it's half-baked anyway.

Reply:
Hm, so the usage info should also come from the markdown instead of `render_usage`?

Reply:
Yeah that one definitely, especially once your other PR for indentation gets merged: https://github.com/uutils/coreutils/pull/4393. If it would come from `render_usage` we'd get additional indentation in the online docs.

Reply:
No, there is already a `trim` in the current implementation (https://github.com/uutils/coreutils/blob/main/src/bin/uudoc.rs#L179) and so #4393 doesn't affect the online docs.

Reply:
Oh right, that one doesn't matter then